.eslintrc

```diff
@@ -25,6 +25,7 @@
     "class-methods-use-this": 0,
     "import/prefer-default-export": 0,
     "no-underscore-dangle": 0,
-    "max-len": 0
+    "max-len": 0,
+    "no-plusplus": 0
```

Comment:
Yes ... I agree .... I like that it is strict but I hate that it is super strict!